### ISSUE TITLE
Google's Permission policy update

### ISSUE THREAD

**Issue by javiramos1** [floatinghotpot/cordova-plugin-sms#83]
Google will not allow reading SMS form January 9th:

https://support.google.com/googleplay/android-developer/answer/9047303

We need to move to this API instead: https://developers.google.com/identity/sms-retriever/overview

I think this plugin won't work from Janary

**Comment by akshgods** [comment]
@javiramos1  right.. any update on this plugin? or any other plugin we can use?

**Comment by freedude** [comment]
Deadline is looming closer, any updates on this plugin?

**Comment by Eskilax** [comment]
Any solution to this problem?
